api/report_server.thrift

```diff
@@ -31,6 +31,8 @@ struct RunData{
   6: string         runCmd,       // the used check command
   7: optional bool  can_delete    // true if codeCheckerDBAccess::removeRunResults()
                                   // is allowed on this run (see issue 151)
+  8: map<shared.DetectionStatus, i32> detectionStatusCount
```

Comment:
There will be a separate count api with filtering if the schema is ready. I'm not sure we should return this value here. For now we can keep it.